compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.h

```diff
@@ -231,6 +233,14 @@ struct AMDAIEOptions {
             clEnumValN(AMDAIEDevice::npu4, "npu4",
                        "Strix B0 NPU with 8 columns and 6 rows")));
 
+    binder.opt<int>("iree-amdaie-num-rows", AMDAIENumRows,
+                    llvm::cl::cat(category),
+                    llvm::cl::desc("Number of rows used in an AIE core array"));
```

Comment:
```suggestion
                    llvm::cl::desc("Number of rows used in an AIE core array. The compiler will choose a tiling strategy that uses no more than this number of rows. However, some workloads (like convolution) currently ignore this flag, and use a hardcoded number of rows."));
```

or something like that, for users without the context